src/app/components/Transcript/index.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable jsx-a11y/aria-role */
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import styles from './index.styles';
+import Text from '../Text';
+import TranscriptTimestamp from './TranscriptTimestamp';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const renderTranscriptItems = (transcriptBlocks: any) =>
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  transcriptBlocks.map((item: any, index: number) => {
+    if (!item) {
+      return null;
+    }
+    const timestamp: string = item?.start;
+    const text: string = item?.content;
+
+    // Remove hours and miliseconds
+    // TO DO - move this to BFF
+    const formattedTimestamp = timestamp.slice(3, -4);
+    return (
+      // eslint-disable-next-line react/no-array-index-key
+      <li key={index} css={styles.listItem}>
+        <Text>
+          <span role="text">
```

Comment:
Lets try this without the role text please

Reply:
I believe I added this in to attempt to get the Timestamp and Transcript text to be announced in one gesture/swipe.

Reply:
But realise it can be applied to the outer Text element and this seems to work fine